lib/topolys/model.rb

```diff
@@ -1121,7 +1121,7 @@ def recalculate
       # check that holes are on same plane as outer
       plane = @outer.plane
       @holes.each do |hole|
```

Comment:
Taking the opportunity for a suggestion. As it stands, Topolys requires that parent applications/wrappers (e.g. TBD) send (as input) holes with wire normals inverted with respect to their parent face. IMO, if a parent application "gets" (i.e. creates) a Topolys face with "holes" (i.e. an array of wires) as input (i.e. the intention is clear enough), Topolys should be able to evaluate if such holes are inverted or not, and invert those that aren't. This looks like internal logic to Topolys, and therefore such functionality (e.g. evaluation, inversion) should - in my opinion - reside within Topolys code. The operation is simple enough (.reverse).

Reply:
... to be clear, I can complete TBD with Topolys as it stands. The suggestion above is for future Topolys considerations.

Reply:
Thanks for the suggestion, I added an issue: https://github.com/automaticmagic/topolys/issues/11